pod/perldelta.pod

```diff
@@ -45,6 +45,35 @@ XXX For a release on a stable branch, this section aspires to be:
 
 [ List each incompatible change as a =head2 entry ]
 
+=head2 Apostrophe is again recognized as a global name separator
+
+This was deprecated in Perl 5.38 and removed as scheduled in perl
+5.41.3, after some discussion has been reinstated by default.
+
+This can be controlled with the C<apostrophe_as_package_separator>
+feature which is enabled by default, but is disabled from the 5.41
+feature bundle onwards.
+
+If you want to disable use within your own code you can explicitly
+disable the feature:
+
+  no feature "apostrophe_as_package_separator";
+
+Note that disabling this feature only prevents use of apostrophe as a
+package separator within code, unlike with the reverted removal
+symbolic references still treat C<'> as C<::> with the feature
+disabled:
```

Comment:
I'm confused by the flow of the previous sentence.  Should it be `unlike with the reverted removal of symbolic references ...`?  Or perhaps, `unlike with the reverted removal, symbolic references ...`

Reply:
removed the mention of the reverted removal